lib/evaluate/characteristics.h

```diff
@@ -145,9 +145,10 @@ struct DummyArgument {
   bool IsOptional() const;
   void SetOptional(bool = true);
   std::ostream &Dump(std::ostream &) const;
-  // name is not a characteristic and so does not participate in operator==
-  // but it is needed to determine if procedures are distinguishable
+  // name and pass are not a characteristics and so does not participate in
```

Comment:
are not ~a~ characteristics

Reply:
Sorry, I missed this comment. I'll fix this in a future change.